src/data/event.py

```diff
@@ -76,10 +76,11 @@ def formatted_text(self) -> str:
                 text=self.text,
             )
         elif self.timer:
+            assert self.timer_hour is not None
             return _('Timer [{timer_uniq_id}]: {text}').format(
                 timer_uniq_id=self.timer_hour.timer.uniq_id, text=self.text
             )
```

Comment:
Since `self.timer_hour` is considered false, this means that it is `None` (an object is considered True unless its class implements a `__bool__()` method that returns `False` or a `__len__()` method that returns zero, when called with the object, see https://docs.python.org/3.12/library/stdtypes.html#truth-value-testing)
```suggestion
            return _('Timer [{timer_uniq_id}]: {text}').format(
                timer_uniq_id=self.timer.uniq_id, text=self.text
            )
```